gcp/api/server.py

```diff
@@ -518,6 +518,28 @@ def to_response(b):
     context.service_context.abort(grpc.StatusCode.INVALID_ARGUMENT,
                                   'Invalid query.')
 
+  if include_details:
```

Comment:
nit: maybe worth adding a comment here to explain why we need to do this separately